packages/desktop/electron/main.js

```diff
@@ -185,6 +185,8 @@ function isUrlAllowed(targetUrl) {
         'explorer.iota.org',
         'iotatreasury.org',
         'govern.iota.org',
+        'explorer.iota-alphanet.iotaledger.net',
+        'explorer.stardust-mainnet.iotaledger.net',
```

Comment:
Is this link needed? Wouldnt it be enough linking to the production explorer `explorer.iota.org`